samples/parselib/src/main/cpp/wav/WavStreamReader.cpp

```diff
@@ -110,48 +110,203 @@ void WavStreamReader::positionToAudio() {
     }
 }
 
-int WavStreamReader::getDataFloat(float *buff, int numFrames) {
-    // __android_log_print(ANDROID_LOG_INFO, TAG, "getData(%d)", numFrames);
+/**
+ * Read and convert samples in PCM8 format to float
+ */
+int WavStreamReader::getDataFloat_PCM8(float *buff, int numFrames) {
+    int numChans = mFmtChunk->mNumChannels;
 
-    if (mDataChunk == 0 || mFmtChunk == 0) {
-        return 0;
+    int buffOffset = 0;
+    int totalFramesRead = 0;
+
+    const static int SAMPLE_SIZE = sizeof(u_int8_t);
+    const static float SAMPLE_FULLSCALE = (float)0x7F;

```

Comment:
Use 0x80.
For unsigned 8 bit samples, 0x80 is baseline.
And we want PCM8 0x00 to map to FLOAT -1.0.

Reply:
Done.